src/main/java/io/dropwizard/kafka/KafkaProducerFactory.java

```diff
@@ -19,16 +19,11 @@
 import org.apache.kafka.common.record.CompressionType;
 import org.hibernate.validator.constraints.NotEmpty;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Optional;
-
 import javax.annotation.Nullable;
 import javax.validation.Valid;
 import javax.validation.constraints.Min;
 import javax.validation.constraints.NotNull;
+import java.util.*;
```

Comment:
Please no * imports

Reply:
Sorry that was the IDE, reverted to explicit imports!